frontend/src/employee-mobile-frontend/components/staff/StaffAttendanceEditor.tsx

```diff
@@ -59,51 +68,72 @@ export default function StaffAttendanceEditor({
       </H2>
       <Subtitle>{i18n.staff.daycareResponsible}</Subtitle>
       <PlusMinus
+        editable={editable}
         value={staff}
         onMinus={dec(staff, setStaff)}
         onPlus={inc(staff, setStaff)}
-        disabled={isSaving}
+        disabled={saving}
         data-qa="staff-count"
       />
-      <Gap size="m" />
+      <Gap size="s" />
       <Subtitle>{i18n.staff.other}</Subtitle>
       <PlusMinus
+        editable={editable}
         value={staffOther}
         onMinus={dec(staffOther, setStaffOther)}
         onPlus={inc(staffOther, setStaffOther)}
-        disabled={isSaving}
+        disabled={saving}
         data-qa="staff-other-count"
       />
-      <Gap size="m" />
-      <FixedSpaceRow justifyContent="center">
-        <Button
-          text={i18n.common.cancel}
-          onClick={reset}
-          disabled={!changed || isSaving}
-          data-qa="cancel-button"
-        />
-        <Button
-          text={i18n.common.confirm}
-          primary
-          disabled={!changed || isSaving}
-          onClick={() =>
-            onConfirm({
-              ...staffAttendance,
-              count: staff,
-              countOther: staffOther
-            })
-          }
-          data-qa="confirm-button"
-        />
+      <Gap size="s" />
+      {groupId && (
+        <>
+          <FixedSpaceRow
+            justifyContent="center"
+            style={{ marginLeft: -32, marginRight: -32 }}
```

Comment:
I was about to write that you shouldn't use inline styles for consistency, but seems like this is not the first place they are used 😄